atom/common/native_mate_converters/v8_value_converter.cc

```diff
@@ -243,12 +238,18 @@ base::Value* V8ValueConverter::FromV8ValueImpl(
     return NULL;
 
   if (val->IsDate()) {
-    if (!date_allowed_)
-      // JSON.stringify would convert this to a string, but an object is more
-      // consistent within this class.
-      return FromV8Object(val->ToObject(), state, isolate);
     v8::Date* date = v8::Date::Cast(*val);
-    return new base::FundamentalValue(date->NumberValue() / 1000.0);
+    v8::Local<v8::Value> toISOString =
+        date->GetRealNamedPropertyInPrototypeChain(
```

Comment:
wasn't sure which would be the best way, this iterates with configuration `PROTOTYPE_CHAIN_SKIP_INTERCEPTOR` while get looksup all the elements. Not aware of benchmarks to which is faster. Have changed it.